app/javascript/crayons/Dropdown/__stories__/Dropdown.stories.jsx

```diff
@@ -5,7 +5,7 @@ import { Dropdown } from '@crayons';
 import './dropdown-css-helper.scss';
 
 export default {
-  title: 'Components/Dropdowns/JSX',
+  title: 'Components/Dropdowns',
```

Comment:
For Preact components, no point on adding another folder in the hierarchy for the Storybook sidebar. It makes sense for HTML components, but not JSX.